pkg/tsdb/cloudwatch/models/cloudwatch_query.go

```diff
@@ -17,22 +17,23 @@ import (
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 
 	"github.com/grafana/grafana/pkg/infra/log"
+	"github.com/grafana/grafana/pkg/tsdb/cloudwatch/kinds/dataquery"
 )
 
 type (
-	MetricEditorMode uint32
-	MetricQueryType  uint32
+	MetricEditorMode dataquery.MetricEditorMode
+	MetricQueryType  dataquery.MetricQueryType
 	GMDApiMode       uint32
 )
 
 const (
-	MetricEditorModeBuilder MetricEditorMode = iota
-	MetricEditorModeRaw
+	MetricEditorModeBuilder = dataquery.CloudWatchMetricsQueryMetricEditorModeN0
```

Comment:
I realize that these were named in a different commit, but this is a terribly named enum

Reply:
yea, i think it's just the way the generator works right now.